### PR TITLE
OSDOCS-2115: Release notes for setting Ingress Controller thread count

### PR DESCRIPTION
**Pull request by skrthomas**
4.8 release note for setting Ingress Controller thread count:
https://issues.redhat.com/browse/OSDOCS-2115
https://issues.redhat.com/browse/OSDOCS-2201

Preview: https://deploy-preview-33262--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-networking

@rfredette @lihongan can you please review this release note and let me know if you have any feedback? Thank you! :)

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -163,6 +163,13 @@ For more information, see xref:../networking/ovn_kubernetes_network_provider/tra
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-converting-http-header-case_configuring-ingress[Converting HTTP header case].
 
+[id="ocp-4-8-setting-ingress-configuring-thread-count"]
+==== Setting Ingress Controller thread count
+
+{product-title} 4.8 adds support for setting the thread count to increase the amount of incoming connections a cluster can handle
```
Comment (lbarbeevargas) [inline release_notes/ocp-4-8-release-notes.adoc:169]: Just missing a period at the end of this sentence. 👍

**Review by lbarbeevargas — COMMENTED** [review]
LGTM! Just one small nit.

**Comment by skrthomas** [comment]
@lihongan when you get a chance can you give this a 👀  🙏

**Comment by rfredette** [comment]
lgtm :+1:

**Comment by lihongan** [comment]
LGTM.
thank you @skrthomas

**Pull request merged**